src/frontend/komponenter/Brevmottakere/BrevMottakere.tsx

```diff
@@ -37,15 +37,22 @@ const Brevmottakere: React.FC<{
     const utledNavnPåMottakere = (brevMottakere: IBrevmottakere) => {
         return [
             ...brevMottakere.personer.map(
-                (person) => `${formatterBeskrivelseAvBrevmottakersRolle(person.mottakerRolle)}`
+                (person) =>
+                    `${mapTilPersonNavnMedKolonEtter(person)}${formatterBeskrivelseAvBrevmottakersRolle(person.mottakerRolle)}`
```

Comment:
```suggestion
                (person) =>
                    `${person?.navn + ':' || ''}${formatterBeskrivelseAvBrevmottakersRolle(person.mottakerRolle)}`
```
`mapTilPersonNavnMedKolonEtter` er unødvendig :)